WordPress/gutenberg
# Test/optional ignore style validation

## What?
Just a rough prototype to see if we can make ignoring styles in validation optional by block.

Related to https://github.com/WordPress/gutenberg/pull/37942

## Why?


## How?


## Testing Instructions

- Add a Spacer block and in the code view add some additional inline styles - switch to editor view and block should remain valid
- Add any other block and modify inline styles in code view and check that block invalidates as normal when switching to editor view

## Screenshots or screencast
